pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -79,13 +79,24 @@ var (
 
 	defaultDestinationRule = networking.DestinationRule{}
 
-	plaintextTransportSocketMatch = &apiv2.Cluster_TransportSocketMatch{
-		Name:  "plaintext",
+	plaintextTransportSocketMatchNoLabel = &apiv2.Cluster_TransportSocketMatch{
+		Name:  "plaintext-nolabel",
 		Match: &structpb.Struct{},
 		TransportSocket: &core.TransportSocket{
 			Name: util.EnvoyRawBufferSocketName,
 		},
 	}
+	plaintextTransportSocketMatchDisabledLabel = &apiv2.Cluster_TransportSocketMatch{
```

Comment:
I'm not quite understand what the differences between these 2. Can the plaintext-nolabel can be used for both no-label and disable-label case?

Reply:
my guess is plaintext-disabled-label is for one way TLS but seems the code isn't updated properly?

Reply:
No.. plaintext disabled is same as label not specified. But i thought I would be explicit

Reply:
Not very convinced that we need both, but it's a minor detail though.